hw_diag/diagnostics/gateway_diagnostics.py

```diff
@@ -0,0 +1,26 @@
+from hm_pyhelper.diagnostics import DiagnosticsReport
+from hm_pyhelper.diagnostics.diagnostic import Diagnostic
+from hw_diag.utilities.miner import fetch_miner_data
+from hm_pyhelper.miner_param import LOGGER
+import grpc
+
+
+class GatewayDiagnostic(Diagnostic):
+    KEY = 'GATEWAY'
+    FRIENDLY_NAME = "gatewayrs"
+
+    def __init__(self):
+        super(GatewayDiagnostic, self).__init__(self.KEY, self.FRIENDLY_NAME)
+
+    def perform_test(self, diagnostics_report: DiagnosticsReport) -> None:
+        diagnostics = {}
+        try:
+            fetch_miner_data(diagnostics)
```

Comment:
It would be less confusing to read if this was `miner_data = fetch_miner_data()`

Reply:
this code no longer exists.